api/client/cli.go

```diff
@@ -160,6 +162,13 @@ func NewDockerCli(in io.ReadCloser, out, err io.Writer, clientFlags *cli.ClientF
 		}
 		cli.configFile = configFile
 
+		// Set the CLI's API version
```

Comment:
Just a suggestion; include something in the comment to describe this is *overriding* the default API version that is used by the client